themes/Night Owl-Light-color-theme-noitalic.json

```diff
@@ -962,6 +962,21 @@
         "foreground": "#aa0982"
       }
     },
+    {
+      "name": "CSS Custom Property declarations",
+      "scope": ["variable.css"],
+      "settings": {
+        "foreground": "#111111"
+      }
+    },
+    {
+				"name": "CSS Custom Property usage",
```

Comment:
The formatting looks a little janky here, can you please fix it? Prettier can help with things like this.

Reply:
weird...I use Prettier :) I will make the change now.

Reply:
How odd, I use Prettier :)

I have updated the formatting in all themes.